test/reporters/xunit.spec.js

```diff
@@ -43,20 +42,20 @@ describe('XUnit reporter', function() {
   });
 
   describe("when 'reporterOptions.output' is provided", function() {
-    var expectedOutput = path.join(path.sep, 'path', 'to', 'some-output');
```

Comment:
Why have you had to remove `path.sep`?

Reply:
I thought it is not necessary and it causes the test to fail.
But now I know it was my misunderstanding. I put it back now.
Thank you for your feedback.

Reply:
I got it finally. It works both ways, with or without `path.sep`. The tests don't check for the value of `expectedOutput`.